formstack-baton-requests/src/test/scala/com/gu/identity/formstackbatonrequests/FormstackPerformSarHandlerSpec.scala

```diff
@@ -0,0 +1,45 @@
+package com.gu.identity.formstackbatonrequests
+
+import com.gu.identity.formstackbatonrequests.aws.SubmissionTableUpdateDate
+import org.scalatest.{FreeSpec, Matchers}
+
+class FormstackPerformSarHandlerSpec extends FreeSpec with Matchers {
+  val mockConfig: PerformSarLambdaConfig =
+    PerformSarLambdaConfig(
+      "resultsBucket",
+      "resultsPath",
+      "encryptionPassword",
+      FormstackAccountToken(1, "accountOneToken"),
+      FormstackAccountToken(2, "accountTwoToken"),
+      "bcryptSalt"
+    )
+
+  "FormstackSarHandler" - {
+    "should update dynamo with submissions from Formstack" in {
+      val lambda =
+        FormstackPerformSarHandler(
+          DynamoClientStub.withSuccessResponse,
+          FormstackClientStub.withSuccessResponse,
+          mockConfig)
+
+      val expectedResponse = Right(())
```

Comment:
are we testing writing nothing to dynamo? should we check with writing some data ?